Removed portacle & links to it

dotemacs opened the pull request:
As per the issue:
https://github.com/LispCookbook/cl-cookbook/issues/587, portacle is no longer maintained, so removing the links to it.

Also took the opportunity and updated the section on how to install Emacs, git, SBCL & Quicklisp on both macOS & Windows.

Not sure where `windows.md` file is linked from. Figured you might be able to point me, if needed.

vindarel commented:
IIRC this windows page isn't linked anymore (but left for old links), we put installation instructions in "getting started", since we can refactor some instructions. A sub-section for MacOS and more for Windows would be welcome.

dotemacs commented:
> thanks. Why, in your opinion, should we remove Portacle? I'd appreciate some discussion. Does it harm users, if we keep it as a solution, but we warn its Emacs version is outdated, and won't give features x y and z ?

My logic was:
- project is stale
- the author won't maintain it
- why give users a stale project to "help" them

I suspect that being given the stale project, won't give much confidence in the book itself.

But again, this is just my view. Happy to go with whatever you decide.

dotemacs commented:
I've revered the update to the `windows.md` file, as per comment above, since it's no longer used.

I've added instructions on how to install Emacs.

I've updated the instructions on how to install `rlwrap` for all three major OSs.

See what you make of it.

Thanks
